packages/eddsa-pcd/test/EDDSAPCD.spec.ts

```diff
@@ -0,0 +1,44 @@
+import "mocha";
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
+/// <reference path="../src/util/declarations/circomlibjs.d.ts" />
+
+import { ArgumentTypeName } from "@pcd/pcd-types";
+import { expect } from "chai";
+import { EdDSAPCD, EdDSAPackage, prove, verify } from "../src";
+
+describe("eddsa-pcd should work", function () {
+  this.timeout(30_000);
+  let pcd: EdDSAPCD;
+
+  it("should be able to sign and verify using the PCD class", async function () {
+    const prvKey =
+      "0001020304050607080900010203040506070809000102030405060708090001";
```

Comment:
how did you create this private key? can it be literally any string? please document that here.